docs/UserGuide.md

```diff
@@ -109,12 +117,17 @@ Example:
 
 Edits a food item from the displayed list of foods.
 
-Format: `edit INDEX [n/FOOD] [c/CALORIES] [t/MEAL_TYPE]`
+Format: `edit INDEX [n/FOOD_NAME] [c/CALORIES] [t/MEAL_TYPE]`
```

Comment:
It would be good to update the commands summary section as well! Same for the `add` and `find` command.

Reply:
ok will do i shall update them!